docs/cli/overview.md

```diff
@@ -0,0 +1,96 @@
+---
+title: .NET Aspire CLI Overview and Commands
+description: Learn .NET Aspire CLI commands for creating projects, running an app host, and adding integrations. Get started with command-line tools to build and manage distributed applications efficiently.
+ms.date: 06/26/2025
+ms.topic: overview
+ms.custom:
+  - ai-gen-docs-bap
+  - ai-gen-title
+  - ai-seo-date:06/26/2025
+  - ai-gen-description
+---
+
+# .NET Aspire CLI Overview
+
+The Aspire CLI (`aspire` command) is a cross-platform tool that provides command-line functionality to create, manage, run, and publish polyglot Aspire projects. Use the Aspire CLI to streamline development workflows and coordinate services for distributed applications.
+
+The Aspire CLI is an interactive-first experience.
+
+- [Install .NET Aspire CLI.](install.md)
```

Comment:
Could we use the fancy green checkbox list style here?

Reply:
Gross